packages/sku/src/lib/configure.ts

```diff
@@ -22,8 +22,13 @@ import { getPathFromCwd, writeFileToCWD } from './cwd.js';
 
 const coverageFolder = 'coverage';
 
-const convertToForwardSlashPaths = (pathStr) => pathStr.replace(/\\/g, '/');
-const addSep = (p) => `${p}${path.sep}`;
+const hasErrorMessage = (e: unknown): e is { message: string } =>
+  (e as { message: string }).message !== undefined;
```

Comment:
Casting inside this function sort of defeats its purpose. It's sort of a 'mostly safe cast' already, so I'd typically do something like checking for the property at runtime instead. It's a little verbose but it's thorough.

```suggestion
const hasErrorMessage = (e: unknown): e is { message: string } =>
  typeof e === 'object' &&
  e !== null &&
  'message' in e &&
  typeof e.message === 'string';
```

Ditto for the error code type guard.